src/main/resources/public/ts/utils/date.utils.ts

```diff
@@ -19,10 +19,6 @@ export class DateUtils {
         return momentDate.format(FORMAT.displayFRDate);
     };
 
-    static getFRDateFormat = (date: Date, parsingFormat?: string): string => {
```

Comment:
i delete the function getFRDateFormat() because this ticket is a fix of another and this function is useless now